Mage.Tests/src/test/java/org/mage/test/combat/AttackPlaneswalkerTest.java

```diff
@@ -127,4 +127,29 @@ public void testAttackPlaneswalkerTriggers() {
         assertPermanentCount(playerB, "Silent Skimmer", 1);
         assertCounterCount("Kiora, the Crashing Wave", CounterType.LOYALTY, 2);
     }
+
+    @Test
+    public void testAttackedPlaneswalkerDestroyed() {
+        addCard(Zone.BATTLEFIELD, playerA, "Kiora, the Crashing Wave");
+        addCard(Zone.BATTLEFIELD, playerA, "Grizzly Bears");
+        addCard(Zone.BATTLEFIELD, playerB, "Grizzly Bears");
+        addCard(Zone.BATTLEFIELD, playerB, "Plains");
+        addCard(Zone.BATTLEFIELD, playerB, "Swamp");
+        addCard(Zone.HAND, playerB, "Despark");
+
+        attack(2, playerB, "Grizzly Bears", "Kiora, the Crashing Wave");
+        castSpell(2, PhaseStep.DECLARE_ATTACKERS, playerB, "Despark");
+        setChoice(playerB, "Kiora, the Crashing Wave");
```

Comment:
Must be `addTarget` command, not `setChoice`. But test works fine (thanks AI choosing). Typo fixed in ee9abe4699650ef2462d730099d7892d409bec98